css/templates/themes/2/page.css

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2017 The AMPStart Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../core';
+@import '../../../../components/footer/footer';
+@import '../../../../components/images/fullpage-hero';
+@import '../../../../components/images/images';
+@import '../../../../components/navbar/navbar';
+@import '../../../../components/social-follow/social-follow';
+@import 'page-vars';
+
+body {
+  line-height: 1.75;
+}
+
```

Comment:
Having to override a variable to keep its value the same when changing a seemingly unrelated variable is pretty convoluted and counterintuitive though. Would you not consider changing the base `--navbar-height` variable so it’s a fixed value which no longer depends on the header variables instead?